ipaclient/install/client.py

```diff
@@ -1370,9 +1370,35 @@ def do_nsupdate(update_txt):
         ipautil.run([paths.NSUPDATE, '-g', UPDATE_FILE])
         result = True
     except CalledProcessError as e:
-        logger.debug('nsupdate failed: %s', str(e))
+        logger.debug('nsupdate (GSS-TSIG) failed: %s', str(e))
         try:
             ipautil.run([paths.NSUPDATE, UPDATE_FILE])
+            try:
+                sssdconfig = SSSDConfig.SSSDConfig()
```

Comment:
It almost feels like this whole sssdconfig part should be in an else:

Reply:
or part of it, but I'm wary of changing the logic there.